src/components/RadioBorderedBox/__tests__/__snapshots__/index.tsx.snap

```diff
@@ -92,15 +92,11 @@ exports[`RadioBorderedBox renders correctly 1`] = `
   opacity: 0.01;
 }
 
-.cache-1sg0ylu.cache-1sg0ylu {
```

Comment:
Intentional ? If I understand corrctly it's not needed anymore because you added the &nbsp to the badgeText ?